plugins/source/okta/client/spec.go

```diff
@@ -10,22 +11,37 @@ import (
 
 type (
 	Spec struct {
-		Token       string     `json:"token,omitempty"`
-		Domain      string     `json:"domain,omitempty"`
-		RateLimit   *RateLimit `json:"rate_limit,omitempty"`
-		Debug       bool       `json:"debug,omitempty"`
-		Concurrency int        `json:"concurrency,omitempty"`
+		// Token for Okta API access.
+		// You can set this with an `OKTA_API_TOKEN` environment variable.
+		Token string `json:"token" jsonschema:"required,minLength=1"`
```

Comment:
This shouldn't be required + this shouldn't impose length restrictions, as the `OKTA_API_TOKEN` is the alternative to set this.
IMO, the correct way is to leave the default representation & doc that empty/missing will result in `OKTA_API_TOKEN` env var check.
```suggestion
		Token string `json:"token"`
```

Reply:
Alt: just drop the env var & make a breaking change, then rebase the code